<launch>
	<node name="pose_to_posestamped" pkg="ros_conversions" type="pose_to_posestamped" output="screen">
			<remap from="pose_to_posestamped/pose_in" to="ground_truth/pose"/>
			<remap from="pose_to_posestamped/posestamped_in" to="abc"/>
			
			<remap from="pose_to_posestamped/pose_out" to="pose_to_posestamped/pose_out"/>
			<remap from="pose_to_posestamped/posestamped_out" to="mavros/local_position/pose"/>
			
			<param name="frame_id" value="world"/>
	</node>
</launch>
